electron/electron/handlers/runWhisper/runWhisper.ts

```diff
@@ -86,21 +86,45 @@ export default ipcMain.handle(
     if (device) inputArray.push('--device', device);
 
     // Add the input path
-    inputArray.push(inputPath);
+    inputArray.push('--input', inputPath);
 
     // ---------------------------------  Run the whisper script --------------------------------- //
 
     console.log('RunWhisper: Running model with args', inputArray);
 
     // Spawn the whisper script
-    const childProcess = spawn('whisper', inputArray, { stdio: 'inherit' });
+    let childProcessArgs: [string, string[]?, object?];
+    if (isDev) {
+      const backendDir = join(__dirname, '../../../../backend/');
+      childProcessArgs = [
+        'poetry',
+        ['run', 'stagewhisper', ...inputArray],
+        {
+          stdio: 'inherit',
+          cwd: backendDir
+        }
+      ];
+    } else {
+      childProcessArgs = ['echo', ['"Production backend not yet implemented"']];
```

Comment:
This will be implemented later once we have the Python build process working (possibly using pyinstaller)